toolkits/global/packages/global-menu/scss/50-components/_menu.scss

```diff
@@ -0,0 +1,112 @@
+/**
+ * Default menu
+ *
+ */
+
+.c-menu {
+	@include u-list-reset();
+	font-size: $menu--font-size;
+	font-family: $menu--font-family;
+	display: flex;
+	line-height: $menu--line-height;
+}
+
+.c-menu__item:not(:last-child) {
+	margin-right: 16px;
```

Comment:
`get-spacing()` ?

Reply:
or a variable?

Reply:
👍 good spot. Have updated to use the variables